src/core/metrics.cc

```diff
@@ -485,7 +490,7 @@ register_ref impl::add_registration(const metric_id& id, const metric_type& type
     if (_value_map.find(name) != _value_map.end()) {
         auto& metric = _value_map[name];
         if (metric.find(rm->info().id.labels()) != metric.end()) {
-            throw double_registration("registering metrics twice for metrics: " + name);
+            throw double_registration("registering metrics twice for metrics: " + name + " " + get_label_as_string(rm->info().id.labels()));
         }
```

Comment:
You can just fmt::format() here. No need to mix formatting with string concatentation.

```c++
fmt::format("registering metrics twice for metric {} with labels {}", ...)
```